Orange/widgets/data/owcolor.py

```diff
@@ -350,6 +350,7 @@ def set_data(self, data):
                                         self._create_proxies(domain.metas))
             self.openContext(data)
             self.data = data.transform(domain)
+            self.data.domain = domain
```

Comment:
... and risk janezd changing it back to `data.transform` the next time he sees this code? If `data.transform` is syntactic sugar it has to have the same behaviour as the thing that this sugar is coating.